src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -7,9 +7,50 @@
 #include "rdmacm_cm_ep.h"
 
 
-static ucs_status_t uct_rdmacm_create_dummy_cq_qp(struct rdma_cm_id *id,
-                                                  struct ibv_cq **cq_p,
-                                                  struct ibv_qp **qp_p)
+void uct_rdmacm_cm_ep_server_connect_cb(uct_rdmacm_cm_ep_t *cep,
+                                        ucs_status_t status)
+{
+    cep->wireup.server.connect_cb(&cep->super.super, cep->user_data, status);
+}
+
+void uct_rdmacm_cm_ep_client_connect_cb(uct_rdmacm_cm_ep_t *cep,
+                                        uct_cm_remote_data_t *remote_data,
+                                        ucs_status_t status)
+{
+    cep->wireup.client.connect_cb(&cep->super.super, cep->user_data,
+                                  remote_data, status);
+}
+
+static UCS_F_ALWAYS_INLINE
+uct_rdmacm_cm_t *uct_rdmacm_cm_ep_get_cm(uct_rdmacm_cm_ep_t *cep)
+{
+    uct_cm_t *cm = ucs_container_of(cep->super.super.iface, uct_cm_t, iface);
```

Comment:
why not just return ucs_container_of(cep->super.super.iface, uct_rdmacm_cm_t, iface);

Reply:
because iface is a member of uct_cm_t, not of uct_rdmacm_cm_t

Reply:
so use super.iface